docs/reference/mapping/fields/id-field.asciidoc

```diff
@@ -16,30 +20,23 @@ PUT my-index-000001/_doc/1
   "text": "Document with ID 1"
 }
 
-PUT my-index-000001/_doc/2?refresh=true
+POST my-index-000001/_doc/?refresh=true
```

Comment:
I think we should undo this change, it feels weird to be mixing auto-generated and custom IDs ?